### ISSUE TITLE
Tests are embedded in dist

### ISSUE THREAD

**Issue by SebastienElet** [SimpliField/sf-preferences#3]
I got error from a production build : 

``` js
ReferenceError: describe is not defined
    at Object.<anonymous> (…/node_modules/sf-preferences/dist/index.js:102:1)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function._load (/usr/lib/node_modules/pm2/node_modules/pmx/lib/transaction.js:62:21)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (…/services/mailer.js:8:19)
    at Module._compile (module.js:410:26)
```

From [the sources](https://github.com/SimpliField/sf-preferences/blob/fe694494e81225aae77c6086cebf4be2d638665e/dist/index.js#L102-L219), tests are embedded in the lib, but mocha is still as devDependency

**Comment by SebastienElet** [comment]
Fixed in d392c64

I published the `1.0.3` version.

**Comment by nfroidure** [comment]
:+1:
